### DISCUSSION TITLE
token access to rows ?

### DISCUSSION THREAD

**Discussion by tavdog** [Budibase/budibase#1366] [Help] [answered]
I'd like to allow for IOT devices to access their configuration via a secret token.  Is this possible or does all access require a user login ?

**Comment by joebudi** [comment, marked as answer]
Hey @tavdog 

Apologies for the late reply. Could you elaborate on this? We are currently working on our new Auth solution at the moment and any additional information would benefit us greatly.

**Reply by joebudi** [reply]
Also, here's our (a few members of the core Budibase team) collective thoughts:

- a common request is to use Budibase as a backend
- we need to do some work around this, including explicitly documenting the process
- we think there might be an opportunity to create a custom API per table/app, allowing you to generate the info for the API call from the builder. For example, as a user, when you click a table, Budibase gives you the URL and request body info etc

Let me know your thoughts on the above.

**Reply by tavdog** [reply]
I got to thinking that I don't really require actual authorization via a token as long as the url is unique enough.  Then the api could just be publicly accessible.
